datasketches/src/frequencies/mod.rs

```diff
@@ -29,7 +29,6 @@ mod serde;
 mod serialization;
 mod sketch;
 
-pub use self::serde::ItemsSerde;
```

Comment:
To @PsiACE,

No need `ItemsSerde` if we already implement serde over concrete `FrequentItemsSketch<i64>` and `FrequentItemsSketch<String>`.

But the C++ impl does have a Serde abstraction you may investigate in https://github.com/apache/datasketches-cpp/blob/master/common/include/serde.hpp

Reply:
cc @AlexanderSaydakov I'd appreciate it if you can share the story and usage of `serde.hpp`.